openlogin/new-v2-user-login/index.config.ts

```diff
@@ -0,0 +1,33 @@
+import path from "path";
+import {
+  PlaywrightTestConfig,
+  PlaywrightWorkerOptions,
+} from "@playwright/test";
+import { TestArgs } from "./index.lib";
+import indexConfig from "../../index.config";
+
+const testUserEmail = `testuser${process.env.GITHUB_JOB}@tor.us`
+
+const projects: Array<
+Pick<PlaywrightWorkerOptions, "browserName"> & Omit<TestArgs, "openloginURL">
+> = [
+  {
+    browserName: "chromium",
+    user: { email: testUserEmail }
```

Comment:
Why creating a separate variable `testUserEmail` instead of putting it directly, it isn't being used elsewhere, right?